src/main/webapp/resources/js/pages/cart/components/CartSamples.jsx

```diff
@@ -72,18 +72,27 @@ function CartSamplesComponent({
   emptyCart,
   displaySample,
   removeSample,
-  removeProject
+  removeProject,
 }) {
-  const filterSamples = e => applyFilter(e.target.value);
+  const s = samples
+    .map((project) => {
+      const { samples, ...p } = project;
+      return project.samples.map((sample) => ({ ...sample, project: p }));
+    })
+    .flat();
+  console.log(s);
```

Comment:
Remove this logging statement and also change the const variable name to make it clear as to what it is

Reply:
Removed in 0b5643b1b98b4b90291a3f3834053255bbd13da7